presentation-insights.ipynb

```diff
@@ -3736,7 +3736,7 @@
    "source": [
     "## Subclasses and Inheritance\n",
     "\n",
-    "Python classes can also inherit from other classes, becoming subclasses - this allows objects which extend or specilaise the classes that they inherit from in the usual object oriented way"
+    "Python classes can also inherit from other classes, becoming subclasses - this allows objects which extend or specialize the classes that they inherit from in the usual object oriented way"
```

Comment:
This one I don't like - 's' is the standard spelling in UK English, with 'z' used by our American cousins.